Bottom: a7d67602d32e8602b938b31b8d3160006bf432d2
Top:    e978dc7ac0adf7cabae561b9be6afc0ff61df552
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-06 02:23:19 +0100

Refresh of remove-hack-in-draw-c-which-br

---

diff --git a/src/draw.c b/src/draw.c
index 262dfec..ff98a15 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -799,10 +799,7 @@ DrawLayerGroup (int group, const BoxType *drawn_area)
           strcmp (Layer->Name, "route") == 0)
         rv = 0;
       if (layernum < max_copper_layer && Layer->On)
-        dapi->draw_layer (Layer, drawn_area, NULL);
-
-      if (gui->gui) /* HACK: Subcomposite each layer in a layer group separately */
-        gui->set_layer (0, group, 0);
+        DrawLayerCommon (Layer, drawn_area, true);
     }
   if (n_entries > 1)
     rv = 1;
